network/tailscale/tailscale.yaml

```diff
@@ -0,0 +1,73 @@
+name: tailscale
+depends:
+  - service: cri
+  - network:
+    - addresses
+    - connectivity
+    - etcfiles
+container:
+  entrypoint: ./containerboot
+  args: []
+  environment:
+  - PATH=/usr/local/bin:/sbin
+  - TS_SOCKET=/var/run/tailscale/tailscaled.sock
+  - TS_STATE_DIR=/var/lib/tailscale
+  - TS_USERSPACE=false
+  - TS_ACCEPT_DNS=true
+  security:
+    maskedPaths: []
+    readonlyPaths: []
+    writeableRootfs: true
+    writeableSysfs: true
```

Comment:
I guess tailscale writes to some `sys` files, usually would only need access `/proc/sys` I beleive